### PR TITLE
Finished plots and figures for reproduction

### PR DESCRIPTION
**Pull request by AlonDaks**

### CODE REVIEW

File: code/stat159lambda/utils/scene_slicer.py [hunk 1]
```diff
@@ -8,13 +8,14 @@
 from stat159lambda.config import NUM_OFFSET_VOLUMES, NUM_VOLUMES
 
 INTEGER_LABELS = {'day-night': {'DAY': 0,
-                                'NIGHT': 1,
-                                'DAWN': 2},
+                                'DAWN': 0,
```
Comment (lisaannyu) [inline code/stat159lambda/utils/scene_slicer.py]: Good catch

File: code/stat159lambda/reproduction/analyze_similarity.py [hunk 2]
```diff
@@ -94,12 +96,32 @@ def save_correlation_percentiles(aggregation):
...
+    image += image.min()
+    outside_brain_mask = np.logical_not(bm.get_brain_mask())[:, :, 24]
+    image[outside_brain_mask] = 0
+    hot_mask = image >= np.percentile(get_correlations('mean', nan=True), 95)
+    hot_pixels = np.zeros_like(image, dtype=np.float)
+    hot_pixels[hot_mask] = image[hot_mask]
+    hot_pixels[~hot_mask] = np.nan
+    plt.imshow(image, interpolation='nearest', cmap='gray')
+    plt.imshow(hot_pixels, interpolation='nearest', cmap='Blues')
+    plot_path = '{0}/figures/correlated_brain.png'.format(REPO_HOME_PATH)
```
Comment (lisaannyu) [inline code/stat159lambda/reproduction/analyze_similarity.py]: Do you want to add plt.colorbar()?  it's not strictly necessary

File: code/stat159lambda/utils/scene_slicer.py [hunk 3]
```diff
@@ -8,13 +8,14 @@
...
+                                'NIGHT': 1
+                                },
                   'int-ext': {'INT': 0,
                               'EXT': 1}}
 
 TUNING_SECONDS_OFFSET = 17
-
+RESPONSE_DELAY = 4
```
Comment (lisaannyu) [inline code/stat159lambda/utils/scene_slicer.py]: Are you using this line anywhere?  If not, you should remove it

Comment (AlonDaks) [inline code/stat159lambda/utils/scene_slicer.py]: Not used -- good catch. this was from an experiment. will remove

File: code/stat159lambda/reproduction/analyze_similarity.py (hunk 2 above)
Comment (AlonDaks) [inline code/stat159lambda/reproduction/analyze_similarity.py]: will add in a PR tomorrow where i brush up plot labels and such

File: code/stat159lambda/config.py [hunk 4]
```diff
@@ -9,16 +9,16 @@
...
 
-NUM_PROCESSES = int(os.environ.get('STAT159_NUM_PROCESSES', 5))
 
 # fMRI data specific constants
-
 NUM_OFFSET_VOLUMES = 9
 
 NUM_VOLUMES = 3543
 
 NUM_VOXELS = 1108800
```
Comment (lisaannyu) [inline code/stat159lambda/config.py]: If you wanted, you could calculate this constant using VOXEL_DIMENSIONS.  But it's fine as it is too

**Pull request merged**